eth-custodian/package.json

```diff
@@ -13,7 +13,7 @@
         "hardhat": "^2.0.8",
         "hardhat-gas-reporter": "^1.0.4",
         "near-api-js": "^0.40.0",
-        "rainbow-bridge": "https://github.com/near/rainbow-bridge",
+        "rainbow-bridge": "https://github.com/near/rainbow-bridge#ede62fcb3614e28e8c5406300381319fec846100",
```

Comment:
Why you depend on a particular version?

Reply:
Rainbow-token-connector [depends](https://github.com/aurora-is-near/rainbow-token-connector/blob/a45e23aed6d3566ea8f89da7d9527da931699fe6/erc20-connector/package.json#L9) on a particular version as well. Moreover, hardhat might fail with a very strange error telling "Missing task argument file" which I figured out can appear when some included contracts are using the newer version of Solidity contract.